### PR TITLE
[185809] Add API for stress test

### PR DESCRIPTION
**Pull request by aDaichiOta**
https://acsmine.tok.access-company.com/redmine/issues/185809

### CODE REVIEW

File: web/controller/stress.ex [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  end
+
+  defp parse_int(str, default) do
+    case Integer.parse(str) do
+      {val, ""} -> val
+      _         -> default
+    end
+  end
+
+  def calc_pi(loop) do
```
Comment (irisTa56) [inline web/controller/stress.ex]: `calc_pi/1` and `sum_list/1` can be private.

Comment (aDaichiOta) [reply web/controller/stress.ex]: Fixed 951472c0e6a7bd019a9a503012a97910cbb984fc

**Comment by aMasakiTakahashi** [comment]
LGTM!

**Comment by ttokit** [comment]
LGTM!

**Comment by irisTa56** [comment]
LGTM!

**Pull request merged**